pkg/webhook/broadcastjob/validating/broadcastjob_create_update_handler.go

```diff
@@ -94,14 +94,20 @@ func validateBroadcastJobSpec(spec *appsv1alpha1.BroadcastJobSpec, fldPath *fiel
 		allErrs = append(allErrs, field.Invalid(fldPath.Root(), spec.Template, fmt.Sprintf("Convert_v1_PodTemplateSpec_To_core_PodTemplateSpec failed: %v", err)))
 		return allErrs
 	}
-	allErrs = append(allErrs, corevalidation.ValidatePodTemplateSpec(coreTemplate, fldPath.Child("template"))...)
 	if spec.Template.Spec.RestartPolicy != v1.RestartPolicyOnFailure &&
 		spec.Template.Spec.RestartPolicy != v1.RestartPolicyNever {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("template").Child("spec").Child("restartPolicy"),
 			spec.Template.Spec.RestartPolicy,
 			"pod restartPolicy can only be Never or OnFailure"))
 	}
-	return allErrs
+	if spec.Template.Labels != nil {
+		if spec.Template.Labels["job-name"] != "" || spec.Template.Labels["controller-uid"] != "" {
```

Comment:
I know it is an existing problem. Can you please replace hardcoded "job-name" and "controller-uid" with constant definition to improve readability?

Reply:
sure, of course